internal/distributedtracing/distributedtracing.go

```diff
@@ -99,7 +101,13 @@ func Init(ctx context.Context, raw []byte, id string) (*otlptrace.Exporter, *tra
 		tlsOption,
 	)
 
+	attrs := make([]attribute.KeyValue, len(distributedTracingConfig.ResourceAttributes))
+	for key, value := range distributedTracingConfig.ResourceAttributes {
+		attrs = append(attrs, attribute.String(key, value))
```

Comment:
Can resource attributes be any key value pair or do we need to do some validation here? For example, they must follow some naming convention, only some defined keys like `service.namespace`, `service.version`, `service.name` and `service.instance.id` can be used etc.

Reply:
It can be any key-value pair. The opentelemetry spec defines some [well-known resource attributes](https://opentelemetry.io/docs/specs/semconv/resource/), but any key is allowed.